packages/tools/src/tools/base/AnnotationTool.ts

```diff
@@ -732,6 +732,17 @@ abstract class AnnotationTool extends AnnotationDisplayTool {
       viewport,
     };
   }
+
+  /**
+   * @param n - array of numbers or a simple number
+   * @returns True if n or the first element of n is finite and not NaN
+   */
+  public static isNumber(n: number[] | number): boolean {
```

Comment:
there is already a isNumber in packages/core/src/utilities/isEqual.ts , adapt that for your use